GoogleService-Info.plist

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>CLIENT_ID</key>
+	<string>706700152757-2dns9lhm9j3b7jbm3hn8l6c26ub3917l.apps.googleusercontent.com</string>
+	<key>REVERSED_CLIENT_ID</key>
+	<string>com.googleusercontent.apps.706700152757-2dns9lhm9j3b7jbm3hn8l6c26ub3917l</string>
+	<key>API_KEY</key>
+	<string>AIzaSyDvMymaerxk1xU-cyKi_EHhHFyIz8tcJu4</string>
+	<key>GCM_SENDER_ID</key>
+	<string>706700152757</string>
+	<key>PLIST_VERSION</key>
+	<string>1</string>
+	<key>BUNDLE_ID</key>
+	<string>com.turing.PupDates</string>
+	<key>PROJECT_ID</key>
+	<string>pupdates-b3204</string>
+	<key>STORAGE_BUCKET</key>
+	<string>pupdates-b3204.appspot.com</string>
+	<key>IS_ADS_ENABLED</key>
+	<false></false>
+	<key>IS_ANALYTICS_ENABLED</key>
+	<false></false>
+	<key>IS_APPINVITE_ENABLED</key>
+	<true></true>
+	<key>IS_GCM_ENABLED</key>
+	<true></true>
+	<key>IS_SIGNIN_ENABLED</key>
+	<true></true>
+	<key>GOOGLE_APP_ID</key>
+	<string>1:706700152757:ios:487760da3dd2472e7193fc</string>
+	<key>DATABASE_URL</key>
+	<string>https://pupdates-b3204.firebaseio.com</string>
+</dict>
+</plist>
```

Comment:
Is this the file that sends the photo to the firebase server?

Reply:
This just provides the information needed to link the firebase storage with the repository.